code/index.html

```diff
@@ -1,22 +1,120 @@
+         <p class="description">Nulla a ullamcorper erat, ut porttitor magna. Phasellus suscipit orci id ornare rutrum.</p>
+      </a>
+      <!--Intstax BUDAPEST 1-->
+      <a href="https://feketekv.hu/" target="_blank">
+         <p class="title"> Budapest</p>
+         <img src="budapestcafé.jpg" alt="Fekete Café in Budapest">
+         <p class="description">Duis porttitor tempus lobortis. Aenean lacinia laoreet nisl, a imperdiet est consectetur quis.</p>
+      </a>
+      <!--Intstax BERLIN 1-->
+      <a href="https://bonanzacoffee.de/wordpress/" target="_blank">
+         <p class="title"> Berlin</p>
+         <img src="berlincafé.jpg" alt="Bonanza Café in Berlin">
+         <p class="description">Aenean facilisis purus sit amet tellus porta, iaculis consequat purus tincidunt. Nam ultricies libero vel sem venenatis auctor.</p>
+      </a>
+      <!--Intstax PORTO 2-->
+      <a href="https://www.tripadvisor.com/Restaurant_Review-g189180-d12448069-Reviews-Mercador_Cafe-Porto_Porto_District_Northern_Portugal.html" target="_blank">
+         <p class="title"> Porto</p>
+         <img src="portocafé.png" alt="Mercador Café in Porto">
+         <p class="description">Nulla a ullamcorper erat, ut porttitor magna. Phasellus suscipit orci id ornare rutrum.</p>
+      </a>
+      <!--Intstax BUDAPEST 2-->
+      <a href="https://feketekv.hu/" target="_blank">
+         <p class="title"> Budapest</p>
+         <img src="budapestcafé.jpg" alt="Fekete Café in Budapest">
+         <p class="description">Duis porttitor tempus lobortis. Aenean lacinia laoreet nisl, a imperdiet est consectetur quis.</p>
+      </a>
+      <!--Intstax BERLIN 2-->
+      <a href="https://bonanzacoffee.de/wordpress/" target="_blank">
+         <p class="title"> Berlin</p>
+         <img src="berlincafé.jpg" alt="Bonanza Café in Berlin">
+         <p class="description">Aenean facilisis purus sit amet tellus porta, iaculis consequat purus tincidunt. Nam ultricies libero vel sem venenatis auctor.</p>
+      </a>
+      <!--Intstax STOCKHOLM 2-->
+      <a href="https://www.dropcoffee.com/" target="_blank">
+         <p class="title"> Stockholm</p>
+         <img src="stockholmcafé.jpg" alt="Drop Café in Stockholm">
+         <p class="description">Praesent aliquam eu magna vel interdum. Nulla sagittis lacinia euismod. In hac habitasse platea dictumst.</p>
+      </a>
+     </section>
+ </section>
+</section>
+
+<!--SOCIAL MEDIA ROW-->
+  <section class="icon-row">
+    <div class="social-menu">
+     <ul>
+         <li> <a href="#"><i class="fa fa-facebook"></i></a></li>
+         <li> <a href="#"><i class="fa fa-twitter"></i></a></li>
+         <li> <a href="#"><i class="fa fa-instagram"></i></a></li>
+         <li> <a href="#"><i class="fa fa-youtube"></i></a></li>
+     </ul>
+    </div>
+</section>
+
+<!--FOOTER-->
+<footer>
+    <a href="#"><img class="footer-logo" src="logo.png" alt="logo" width="25px"> © Espresso Crema 2020 </a>
+</footer>
```

Comment:
Don't forget to close the img-tag at the end! And try to keep all styling in the CSS-file

Reply:
Definitely, thank you!